packages/eslint-config/.eslintrc.js

```diff
@@ -0,0 +1,49 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/eslint-config
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  plugins: ['eslint-plugin', '@typescript-eslint'],
+  env: {
+    es6: true,
+    node: true,
+  },
+  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
+  rules: {
```

Comment:
_Re-posting https://github.com/strongloop/loopback-next/pull/2492#discussion_r263713318_

I feel it's important to understand which rules enabled by our current configuration will be lost after your change. Quoting from an earlier comment:

> Have you verified that all rules we are currently using in `@loopback/tslint-config` already have their counterpart in `@typescript-eslint` or other eslint plugins?
> 
> If there are any rules that are not available yet, then we should have a discussion whether we are ok to accept that limitation.

The current TSLint config is based on the rules recommended at the time we created them. What's more important, it contains rules that have already helped us to detect important bad practices in the past. I don't want to loose them.

What I would like to see:
 - List of rules that are enabled now but won't be applied after your change
 - If there are any GH issues in eslint/tslint projects that are tracking these missing rules, then please reference them in the list.
 - The PR description should be updated with this information.

Reply:
FYI: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md provides a comprehensive list of linting rules with their implementation status.

Reply:
@raymondfeng PTAL ☝️ 

> I feel it's important to understand which rules enabled by our current configuration will be lost after your change.

Reply:
Updated.